engine/api/workflow/execute_node_run.go

```diff
@@ -347,9 +347,10 @@ func executeNodeRun(ctx context.Context, db gorp.SqlExecutor, store cache.Store,
 				log.Error(ctx, "workflow.execute> Unable to load mutex-locked workflow rnode un: %v", errWRun)
 				return report, nil
 			}
-			AddWorkflowRunInfo(workflowRun, false, sdk.SpawnMsg{
+			AddWorkflowRunInfo(workflowRun, sdk.SpawnMsg{
```

Comment:
i think we could have a nicer constructor like `sdk.NewSpwnMsg(sdk.MsgWorkflowNodeMutexRelease, waitingRun.WorkflowNodeName)`

Reply:
#5049   I will make another PR for this